airflow/jobs/scheduler_job.py

```diff
@@ -853,6 +858,10 @@ def _run_scheduler_loop(self) -> None:
         # Check on start up, then every configured interval
         self.adopt_or_reset_orphaned_tasks()
 
+        if self._task_queued_timeout:
+            self._fail_tasks_stuck_in_queued()
+            timers.call_regular_interval(self._task_queued_timeout, self._fail_tasks_stuck_in_queued)
```

Comment:
Do you need both of these lines? Don't you only want the one wrapped in `timers.call_regular_interval`

Reply:
This probably doesn't need to run at startup. Adoption makes sense to do because it's pretty likely that another scheduler just shut down if we have a new one starting, but I don't we have a similar situation here.